src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -290,40 +290,50 @@ namespace Microsoft.FSharp.Collections
         ///
         /// <example id="compare-with-1">
         /// <code lang="fsharp">
-        /// ([1; 2], [1; 2])
-        /// ||> Seq.compareWith (fun a b -> a.CompareTo(b))
+        /// let closerToNextDozen a b =
```

Comment:
This was asked in the previous PR, I followed these patterns for other functions, making slightly more complex examples in cases like `foldBack` to showcase the potential and hopefully not swamping it too much.